### PR TITLE
Pulumi 3.48.0

### PR DESCRIPTION
**Pull request by t0yv0**
Update Pulumi dependency to keep up with code generation changes.

### CODE REVIEW

File: sdk/nodejs/randomShuffle.ts [hunk 1]
```diff
@@ -24,11 +24,7 @@ import * as utilities from "./utilities";
  *     ],
  *     resultCount: 2,
  * });
- * const example = new aws.elb.LoadBalancer("example", {
- *     // Place the ELB in any two of the given availability zones, selected
```
Comment (lblackstone) [inline sdk/nodejs/randomShuffle.ts:28]: Any idea why the comments are getting dropped in these examples?

**Review by lblackstone — APPROVED** [review]

File: sdk/nodejs/randomShuffle.ts (hunk 1 above)
Comment (t0yv0) [reply sdk/nodejs/randomShuffle.ts:28]: Umm, no. Something is fishy and the comment is only preserved in YAML. I'll file an issue in pulumi/pulumi.

```
jq -r .resources["random:index/randomShuffle:RandomShuffle"].description provider/cmd/pulumi-resource-random/schema.json
```

```
The resource `random.RandomShuffle` generates a random permutation of a list of strings given as an argument.

{{% examples %}}
## Example Usage
{{% example %}}

```typescript
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";
import * as random from "@pulumi/random";

const az = new random.RandomShuffle("az", {
    inputs: [
        "us-west-1a",
        "us-west-1c",
        "us-west-1d",
        "us-west-1e",
    ],
    resultCount: 2,
});
const example = new aws.elb.LoadBalancer("example", {availabilityZones: [az.results]});
// ... and other aws_elb arguments ...
```
```python
import pulumi
import pulumi_aws as aws
import pulumi_random as random

az = random.RandomShuffle("az",
    inputs=[
        "us-west-1a",
        "us-west-1c",
        "us-west-1d",
        "us-west-1e",
    ],
    result_count=2)
example = aws.elb.LoadBalancer("example", availability_zones=[az.results])
# ... and other aws_elb arguments ...
```
```csharp
using System.Collections.Generic;
using Pulumi;
using Aws = Pulumi.Aws;
using Random = Pulumi.Random;

return await Deployment.RunAsync(() => 
{
    var az = new Random.RandomShuffle("az", new()
    {
        Inputs = new[]
        {
            "us-west-1a",
            "us-west-1c",
            "us-west-1d",
            "us-west-1e",
        },
        ResultCount = 2,
    });

    var example = new Aws.Elb.LoadBalancer("example", new()
    {
        AvailabilityZones = new[]
        {
            az.Results,
        },
    });

    // ... and other aws_elb arguments ...
});
```
```go
package main

import (
	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/elb"
	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		az, err := random.NewRandomShuffle(ctx, "az", &random.RandomShuffleArgs{
			Inputs: pulumi.StringArray{
				pulumi.String("us-west-1a"),
				pulumi.String("us-west-1c"),
				pulumi.String("us-west-1d"),
				pulumi.String("us-west-1e"),
			},
			ResultCount: pulumi.Int(2),
		})
		if err != nil {
			return err
		}
		_, err = elb.NewLoadBalancer(ctx, "example", &elb.LoadBalancerArgs{
			AvailabilityZones: pulumi.StringArray{
				az.Results,
			},
		})
		if err != nil {
			return err
		}
		return nil
	})
}
```
```java
package generated_program;

import com.pulumi.Context;
import com.pulumi.Pulumi;
import com.pulumi.core.Output;
import com.pulumi.random.RandomShuffle;
import com.pulumi.random.RandomShuffleArgs;
import com.pulumi.aws.elb.LoadBalancer;
import com.pulumi.aws.elb.LoadBalancerArgs;
import java.util.List;
import java.util.ArrayList;
import java.util.Map;
import java.io.File;
import java.nio.file.Files;
import java.nio.file.Paths;

public class App {
    public static void main(String[] args) {
        Pulumi.run(App::stack);
    }

    public static void stack(Context ctx) {
        var az = new RandomShuffle("az", RandomShuffleArgs.builder()        
            .inputs(            
                "us-west-1a",
                "us-west-1c",
                "us-west-1d",
                "us-west-1e")
            .resultCount(2)
            .build());

        var example = new LoadBalancer("example", LoadBalancerArgs.builder()        
            .availabilityZones(az.results())
            .build());

    }
}
```
```yaml
resources:
  az:
    type: random:RandomShuffle
    properties:
      inputs:
        - us-west-1a
        - us-west-1c
        - us-west-1d
        - us-west-1e
      resultCount: 2
  example:
    type: aws:elb:LoadBalancer
    properties:
      # Place the ELB in any two of the given availability zones, selected
      #   # at random.
      availabilityZones:
        - ${az.results}
```
{{% /example %}}
{{% /examples %}}
```

File: provider/go.mod [hunk 2]
```diff
@@ -3,8 +3,8 @@ module github.com/pulumi/pulumi-random/provider/v4
 go 1.18
 
 require (
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.28.0
-	github.com/pulumi/pulumi/sdk/v3 v3.38.0
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.33.1-0.20221104131001-330791e758b8
```
Comment (t0yv0) [inline provider/go.mod:6]: Hold on, this may have been accidental, I did not intend to reference a pre-release version.

Comment (t0yv0) [reply provider/go.mod:6]: Ah. We're getting this by updating to pulumi/pkg v3.48.0 indirectly which depends on this version of the bridge.

File: sdk/nodejs/randomShuffle.ts (hunk 1 above)
Comment (t0yv0) [reply sdk/nodejs/randomShuffle.ts:28]: https://github.com/pulumi/pulumi/issues/11505

**Pull request merged**